coffee_roaster_market/apps/item/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from coffee_roaster_market.apps.coffee.models import CoffeeModel
+
+
+class ItemModel(models.Model):
+    class Meta:
+        db_table = "item"
+
+    coffee = models.OneToOneField(CoffeeModel, on_delete=models.CASCADE)
```

Comment:
As mentioned during phone call - `ForeignKey` will be more sufficient (Many to one relationship)

Reply:
Done